src/app/components/socialSection/socialSection.module.css

```diff
@@ -0,0 +1,35 @@
+.socialSection {
+  display: flex;
+  gap: 4rem;
+  align-items: center;
+  padding: 60px 60px 60px 0;
+}
+
+.socialSection p {
+  font-family: 'Raleway', sans-serif;
```

Comment:
I'm thinking this could be defined using CSS variables, along with other global styles like specific theme colors, etc.

Reply:
I think this is a great idea. Lets create another ticket for adding global css variables to tackle consistency across the web app.